test/unit/rules/no-yield-to-default-test.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+const ERROR_MESSAGE = require('../../../lib/rules/no-yield-to-default').ERROR_MESSAGE;
+
+generateRuleTests({
+  name: 'no-yield-to-default',
+
+  config: true,
+
+  good: [
+    '{{yield}}',
+    '{{has-block}}',
+    '{{has-block-params}}',
+    '{{hasBlock}}',
+    '{{hasBlockParams}}',
```

Comment:
Can you add a few more "good" scenarios for other expected invocations of these keywords (e.g. passing the expected block names)?

Reply:
done